pages/linux/ufw.md

```diff
@@ -11,14 +11,26 @@
 
 `ufw disable`
 
-- Add ufw allow rule:
+- Show ufw rules along with their numbers:
 
-`ufw allow {{port}} {{service_name}}`
+`ufw status numbered`
 
-- Add ufw deny rule:
+- Allow incoming traffic on port 5432 on this host:
 
-`ufw deny {{port}} {{service_name}}`
+`ufw allow {{5432}}`
 
-- Show ufw rules:
+- Allow traffic from 192.168.0.4 to any address on this host on port 22 using only tcp protocol:
```

Comment:
"using only tcp protocol" --> "using only the TCP protocol". Also, commas after "host" and "22".